HW1/P6/P6.py

```diff
@@ -0,0 +1,142 @@
+# (c) L.Spiegelberg 2015
+import findspark
+findspark.init()
+from pyspark import SparkContext, SparkConf
+
+import urllib
+import numpy as np
+from scipy import stats
+import re
+import os
+import sys
+
+
+def downloadFiles():
+	# load textlist from net
+	filename = "pg100.txt"
+
+	url = "http://www.gutenberg.org/cache/epub/100/pg100.txt"
+
+	wordfile = urllib.URLopener()
+	wordfile.retrieve(url, filename)
+
+	if not os.path.isfile('Shakespeare.txt'):
+		# the texts from shakespeare are contained in lines 182 - 124367
+		# create the file Shakespeare.txt using the line range from above
+		# if all is ok, this should return True, True
+		with open(filename, 'r') as f:
+		    lines = f.readlines()
+		    lines = lines[181:124367]
+		    with open('Shakespeare.txt', 'w') as f2:
+		        f2.writelines(lines)
+		    print f2.closed
```

Comment:
This seems like a debugging thing that should be removed from final code, since "with open" automatically closes the file. Maybe change it to an assert instead, so it doesn't print output? This is a very minor nitpick, though.